### PR TITLE
Adds CHANGELOG

### PR DESCRIPTION
**Pull request by gopalindians**
See:#1233 

__Whats covered__

- All the releases according to [keepchangelog.com](https://keepachangelog.com/en/1.0.0/)
- All the compare links
- All the date accordingly

__Whats not covered__

- Unreleased is not included[needs help]

### CODE REVIEW

**Comment by pkarw** [comment]
Thanks! Could You please extend this change log adding links OR just pasting the upgrade notes? I'm not sure if it's a good practice but for sure it would be useful

**Comment by gopalindians** [comment]
@pkarw Could you please specify which links need to add here and also its good practice if we separate UPGRADE notes in its own file. We can add a hyperlink to UPGRADE notes in CHANGELOG.

**Comment by pkarw** [comment]
Ok, we can do that. I meant links to upgrade notes.

@igloczek could you do a CR of this please?

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# Changelog
```
Comment (Igloczek) [inline CHANGELOG.md]: This part should be same as source

```
# Changelog
All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).

## [Unreleased]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+
+## [1.0.5] - 2018-06-04
+### Fixed
+- Minor fixes
```
Comment (Igloczek) [inline CHANGELOG.md]: Oh, come on :) Let's at least copy commit messages.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+[1.0.3]: https://github.com/DivanteLtd/vue-storefront/compare/v1.0.2...v1.0.3
+[1.0.2]: https://github.com/DivanteLtd/vue-storefront/compare/v1.0.1...v1.0.2
+[1.0.1]: https://github.com/DivanteLtd/vue-storefront/compare/v1.0.0...v1.0.1
+[1.0.0]: https://github.com/DivanteLtd/vue-storefront/compare/v1.0.0-rc.3...v1.0.0
+[1.0.0-rc.3]: https://github.com/DivanteLtd/vue-storefront/compare/v1.0.0-rc.2...v1.0.0-rc.3
+[1.0.0-rc.2]: https://github.com/DivanteLtd/vue-storefront/compare/v1.0.0-rc.0...v1.0.0-rc.2
+[1.0.0-rc.0]: https://github.com/DivanteLtd/vue-storefront/compare/v0.4.0...v1.0.0-rc.0
+[0.4.0]: https://github.com/DivanteLtd/vue-storefront/compare/v0.3.0...v0.4.0
+[0.3.0]: https://github.com/DivanteLtd/vue-storefront/compare/v0.2.1-alpha.0...v0.0.3
+[0.2.1-alpha.0]: https://github.com/DivanteLtd/vue-storefront/compare/v0.2.0-alpha.0...v0.2.1-alpha.0
```
Comment (Igloczek) [inline CHANGELOG.md]: Newline missing

File: CHANGELOG.md (hunk 2 above)
Comment (gopalindians) [reply CHANGELOG.md]: for [1.0.5](https://github.com/DivanteLtd/vue-storefront/releases/tag/v1.0.5) and [1.0.0](https://github.com/DivanteLtd/vue-storefront/releases/tag/v1.0.0) the releases link showing same, I have added for `1.0.0`  but not for `1.0.5`

File: CHANGELOG.md (hunk 1 above)
Comment (gopalindians) [reply CHANGELOG.md]: Sure!

File: CHANGELOG.md (hunk 3 above)
Comment (gopalindians) [reply CHANGELOG.md]: Thanks

**Comment by gopalindians** [comment]
@pkarw I have added [0.3.0](https://github.com/DivanteLtd/vue-storefront/releases?after=v0.4.0) based on releases page, should we follow the release page or [@piotrkarwatka](https://medium.com/@piotrkarwatka)?

**Comment by pkarw** [comment]
Version list seems like complete - I was just concern about the dates; as far as I remember we did release a version in Jan/Feb ... The dates are mismatched in version history because some mess with the tags at some point (we renamed the version tags to be semantic at one day and it messed up the dates :))

**Comment by gopalindians** [comment]
There is another pull request(#1267) for the same with the messed up Dates. Is there any way we can fix this?

**Comment by pkarw** [comment]
@gopalindians should we close this PR and merge in #1267 ?

**Comment by gopalindians** [comment]
@pkarw  Sure

**Pull request closed without merging**